ssh-keygen.c

```diff
@@ -3773,6 +3791,13 @@ main(int argc, char **argv)
 			    &private, attest);
 			if (r == 0)
 				break;
+			if (r == SSH_ERR_KEY_BAD_PERMISSIONS &&
+			    (sk_flags & SSH_SK_RESIDENT_KEY) != 0 &&
+			    (sk_flags & SSH_SK_FORCE_OPERATION) == 0 &&
+			    confirm_sk_overwrite(sk_application, sk_user)) {
+				sk_flags |= SSH_SK_FORCE_OPERATION;
+				continue;
```

Comment:
make remove the continue and make the next statement an `else if` to reuse the below printf reminder?

I notice that this block also messes up the use of `i` as the number of incorrect passphrase prompts - maybe we need to move the `i++` from the `for()` loop to the actual case where the password was incorrect?